xds/src/main/java/io/grpc/xds/LocalityStore.java

```diff
@@ -108,46 +103,24 @@ abstract LocalityStore newLocalityStore(
     private final Helper helper;
     private final LoadBalancerProvider loadBalancerProvider;
     private final ThreadSafeRandom random;
-    private final LoadStatsStore loadStatsStore;
-    private final OrcaPerRequestUtil orcaPerRequestUtil;
```

Comment:
I am removing the integration of orca utilities. They are not used, and even the new design architecture does not consider using it. They are effectively dead code. There is not reason to keep them there. They only make the implementation hard to read and test.